modules/fedimint-ln-server/src/lib.rs

```diff
@@ -1160,3 +1170,264 @@ mod fedimint_migration_tests {
         .await;
     }
 }
+
+mod migrations {
+    use std::io::Error;
+
+    use fedimint_core::db::DatabaseTransaction;
+    use fedimint_core::encoding::{Decodable, DecodeError, Encodable};
+    use fedimint_core::module::registry::ModuleDecoderRegistry;
+    use fedimint_core::{impl_db_lookup, impl_db_record, OutPoint, PeerId};
+    use fedimint_ln_common::contracts::incoming::{
+        FundedIncomingContract, IncomingContract, IncomingContractOffer,
+    };
+    use fedimint_ln_common::contracts::{
+        outgoing, ContractId, DecryptedPreimage, EncryptedPreimage, FundedContract,
+        PreimageDecryptionShare,
+    };
+    use fedimint_ln_common::db::{
+        AgreedDecryptionShareKey, ContractKey, OfferKey, ProposeDecryptionShareKey,
+    };
+    use fedimint_ln_common::ContractAccount;
+    use futures::stream::StreamExt;
+    use serde::{Deserialize, Serialize};
+
+    pub async fn migration_v0(dbtx: &mut DatabaseTransaction<'_>) -> anyhow::Result<()> {
```

Comment:
Let's add a comment here on what the db migration is doing (i.e which structs changed encodings). This is to remove the dependency on `bincode` right?